src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_HostObjectMemberReference.cs

```diff
@@ -12,7 +10,7 @@ internal partial class LocalRewriter
         public override BoundNode VisitHostObjectMemberReference(BoundHostObjectMemberReference node)
         {
             Debug.Assert(_previousSubmissionFields != null);
-            Debug.Assert(!_factory.CurrentMethod.IsStatic);
+            Debug.Assert(!_factory.CurrentMethod.GetContainingMember().IsStatic);
```

Comment:
Maybe worth changing `CurrentMethod` to `CurrentMethodOrNestedFunction` to match style from other code.

Reply:
Should I go ahead and make this change?